chiaki4deck@1.6.6: download via aria2 failed

chiaki4deck didn't ship the MSYS2 version for version 1.6.6 due to a bug with rumble. See related: https://github.com/streetpea/chiaki4deck/issues/307

I've made a PR linked to this to resolve the issue and download the VC version instead.

Console output just in case it's needed:
```
Updating 'chiaki4deck' (1.6.5 -> 1.6.6)
Downloading new version
Starting download with aria2 ...
Download: 04/08 17:36:27 [ERROR] CUID#8 - Download aborted. URI=https://github.com/streetpea/chiaki4deck/releases/downloDownload: Exception: [AbstractCommand.cc:351] errorCode=3 URI=https://github.com/streetpea/chiaki4deck/releases/downloadDownload: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 4cfc56|ERR |       0B/s|C:/Users/Sn0wCrack/scoop/cache/chiaki4deck#1.6.6#https_github.com_streetpea_chiaki4deck_releases_download_v1.6.6_Chiaki4deck-win_x64-MSYS2-Release.zip
Download: Status Legend:
Download: (ERR):error occurred.
Download: aria2 will resume download if the transfer is restarted.
Download: If there are any errors, then see the log file. See '-l' option in help/man page for details.

ERROR Download failed! (Error 3) Resource was not found
ERROR https://github.com/streetpea/chiaki4deck/releases/download/v1.6.6/Chiaki4deck-win_x64-MSYS2-Release.zip
    referer=https://github.com/streetpea/chiaki4deck/releases/download/v1.6.6/
    dir=C:\Users\Sn0wCrack\scoop\cache
    out=chiaki4deck#1.6.6#https_github.com_streetpea_chiaki4deck_releases_download_v1.6.6_Chiaki4deck-win_x64-MSYS2-Release.zip

ERROR & 'C:\Users\Sn0wCrack\scoop\apps\aria2\current\aria2c.exe' --input-file='C:\Users\Sn0wCrack\scoop\cache\chiaki4deck.txt' --user-agent='Scoop/1.0 (+http://scoop.sh/) PowerShell/7.4 (Windows NT 10.0; Win64; x64; Core)' --allow-overwrite=true --auto-file-renaming=false --retry-wait=2 --split=5 --max-connection-per-server=5 --min-split-size=5M --console-log-level=warn --enable-color=false --no-conf=true --follow-metalink=true --metalink-preferred-protocol=https --min-tls-version=TLSv1.2 --stop-with-process=34564 --continue --summary-interval=0 --auto-save-interval=1
```

--- a/bucket/chiaki4deck.json
+++ b/bucket/chiaki4deck.json
@@ -8,7 +8,7 @@
             "url": "https://github.com/streetpea/chiaki4deck/releases/download/v1.6.6/Chiaki4deck-win_x64-VC-Release.zip",
             "hash": "c16475f05bd55ac8b38c226646bbfe64fd5dbc7e24b352215dadb46473a1c561",
             "pre_install": [
-                "$archive = \"Chiaki4deck-Win_VC-$version.zip\"",
+                "$archive = \"Chiaki4deck-Win_x64-VC-$version.zip\"",
                 "Expand-7zipArchive \"$dir\\$archive\" \"$dir\" -ExtractDir \"Chiaki4deck-VC\"",
                 "Remove-Item \"$dir\\$archive\""
             ]

--- a/bucket/chiaki4deck.json
+++ b/bucket/chiaki4deck.json
@@ -5,11 +5,11 @@
     "license": "AGPL-3.0-only",
     "architecture": {
         "64bit": {
-            "url": "https://github.com/streetpea/chiaki4deck/releases/download/v1.6.6/Chiaki4deck-win_x64-MSYS2-Release.zip",
-            "hash": "4e6000fa6b5081d842dfac72f356aa5ffcb06e51c6b498a5aa9101fae403c513",
+            "url": "https://github.com/streetpea/chiaki4deck/releases/download/v1.6.6/Chiaki4deck-win_x64-VC-Release.zip",
+            "hash": "c16475f05bd55ac8b38c226646bbfe64fd5dbc7e24b352215dadb46473a1c561",
             "pre_install": [
-                "$archive = \"Chiaki4deck-Win.zip\"",
-                "Expand-7zipArchive \"$dir\\$archive\" \"$dir\" -ExtractDir \"Chiaki4deck-Win\"",
+                "$archive = \"Chiaki4deck-Win_VC-$version.zip\"",
+                "Expand-7zipArchive \"$dir\\$archive\" \"$dir\" -ExtractDir \"Chiaki4deck-VC\"",
                 "Remove-Item \"$dir\\$archive\""
             ]
         }
@@ -32,7 +32,7 @@
     "autoupdate": {
         "architecture": {
             "64bit": {
-                "url": "https://github.com/streetpea/chiaki4deck/releases/download/v$version/Chiaki4deck-win_x64-MSYS2-Release.zip"
+                "url": "https://github.com/streetpea/chiaki4deck/releases/download/v$version/Chiaki4deck-win_x64-VC-Release.zip"
             }
         },
         "extract_dir": "chiaki4deck_$version"

